From 624dadf2c300d9e5e439c07e1244babcef2fedca Mon Sep 17 00:00:00 2001
From: Holger Freyther <zecke@openmoko.org>
Date: Tue, 13 May 2008 18:53:40 +0100
Subject: [PATCH] Re: [PATCH 6/7] fix-suppress-cpu-suspend-save-restore-messages.patch

Andy Green <andy@...> writes:


I kind of dislike commenting out code. Is this any better?
---
 arch/arm/plat-s3c24xx/pm.c |    9 ++++++++-
 1 files changed, 8 insertions(+), 1 deletions(-)

diff --git a/arch/arm/plat-s3c24xx/pm.c b/arch/arm/plat-s3c24xx/pm.c
index 4fdb311..ae8efaf 100644
--- a/arch/arm/plat-s3c24xx/pm.c
+++ b/arch/arm/plat-s3c24xx/pm.c
@@ -169,8 +169,15 @@ static void s3c2410_pm_debug_init(void)
 }
 
 #define DBG(fmt...) pm_dbg(fmt)
+#define RESTORE_DBG(fmt...) printk(KERN_DEBUG fmt)
 #else
+#if 0
 #define DBG(fmt...) printk(KERN_DEBUG fmt)
+#define RESTORE_DBG(fmt...) printk(KERN_DEBUG fmt)
+#else
+#define DBG(fmt...) do { } while (0)
+#define RESTORE_DBG(fmt...) do { } while (0)
+#endif
 
 #define s3c2410_pm_debug_init() do { } while(0)
 
@@ -392,7 +399,7 @@ void s3c2410_pm_do_save(struct sleep_save *ptr, int count)
 void s3c2410_pm_do_restore(struct sleep_save *ptr, int count)
 {
 	for (; count > 0; count--, ptr++) {
-		printk(KERN_DEBUG "restore %p (restore %08lx, was %08x)\n",
+		RESTORE_DBG("restore %p (restore %08lx, was %08x)\n",
 		       ptr->reg, ptr->val, __raw_readl(ptr->reg));
 
 		__raw_writel(ptr->val, ptr->reg);
-- 
1.5.6.5

